src/mad_spark_alt/cli.py

```
@@ -668,6 +721,15 @@
                 console.print(
                     f"• Best from generation: {metrics.get('best_fitness_generation', 0)}"
                 )
+                
+                # Show semantic operator usage if enabled
+                if metrics.get('semantic_operators_enabled', False):
+                    console.print(f"\n[green]🧬 Semantic Operators:[/green]")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)